CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # splat Release Notes
 
+### 0.22.0: Palette Revamp
+
+* The N64 ci/palette system has been rewritten to be more versatile and support a larger variety of configurations.
+  * ci segments now have a "palettes:" argument, which can be a list of palettes or a single palette to be linked to the ci for extraction. Each palette linked to a ci will result in a separate png
+  * palette segments can provide a "pal_id" field, which serves as a globally searchable palette id. This can be used for cross-segment ci/palette linking.
+  * For more information, see the docs! TODO
+* `spimdisasm` 1.21.0 or above is now required.
```

Comment:
I think it should explicitly specify that the old `raster_name` and `palette` no longer works and what replaces it

Reply:
oh true, I forgot those fields. thanks